### ISSUE TITLE
feat: Add information on the graphs in the List feature

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/docs#236]
Right now, our page for the List feature needs updated. Luckily, our Maintainer Guide has more information.

- Use [this information](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#lists-connecting-with-contributors) to update the List feature section. 

## Tips
- Use the list feature so you have an understanding of what you're looking at.
- If you have questions about the feature, leave them in the comments of this issue
- Adding docs to this section should be targeted at a general audience. That means you can't take the information from the guide word for word. Think about maintainers, contributors, and organizations. 
- Use the images in the guide, rather than creating new ones.

**Comment by v-aryan** [comment]
Hey @BekahHW , from my limited knowledge after reading what has to be done. Adding content from Maintainers guide to list feature ,  listing the subtitles which should be included into list feature are:

-> Lists: Connecting with Contributors
-> Why Create a List?
-> Creating a New List
-> Using Your List

adding these titles with content and images from maintainers guide!

**Comment by Lymah123** [comment]
Hi @BekahHW , is the issue still available?

**Comment by BekahHW** [comment]
@v-aryan yes, those should be subtitles. The difference here is that this portion of the docs should be generally applicable for contributors and maintainers. 

Do you want to take this issue? Sorry for the late reply. I somehow missed this.

**Comment by Lymah123** [comment]
@BekahHW , any update on this issue?

**Comment by BekahHW** [comment]
@Lymah123 do you have any questions on this?

**Comment by Lymah123** [comment]
> @Lymah123 do you have any questions on this?

Hi @BekahHW! I just want to start working on it. I will let you know if I have any questions. Thank you!

**Comment by CBID2** [comment]
Hey @BekahHW. Since this feature has been deprecated, I think we should close this issue. What do you think?

**Comment by BekahHW** [comment]
@CBID2 the list feature is not deprecated. It's renamed to contributor insights. Lymah already has a PR up that we've been working through for it with the changes.

**Comment by CBID2** [comment]
> @CBID2 the list feature is not deprecated. It's renamed to contributor insights. Lymah already has a PR up that we've been working through for it with the changes. 

Ahh ok @BekahHW
